agent/command/s3_put_test.go

```diff
@@ -36,21 +36,6 @@ func TestS3PutValidateParams(t *testing.T) {
 
 			cmd = &s3put{}
 
-			Convey("a missing aws key should cause an error", func() {
-
-				params := map[string]any{
-					"aws_secret":   "secret",
-					"local_file":   "local",
-					"remote_file":  "remote",
-					"bucket":       "bck",
-					"permissions":  "public-read",
-					"content_type": "application/x-tar",
-					"display_name": "test_file",
-				}
-				err := cmd.ParseParams(params)
-				require.Error(t, err)
-				So(err.Error(), ShouldContainSubstring, "AWS key cannot be blank")
-			})
```

Comment:
The deleted tests in this file are covered in s3Operations. Unlike s3.get, I didn't rewrite the remaining tests since it was taking a while to get this PR out